Frameworks/CoreGraphics/CGBitmapContext.mm

```diff
@@ -84,16 +88,21 @@ CGImageAlphaInfo CGBitmapContextGetAlphaInfo(CGContextRef context) {
     }
 
     return (CGImageAlphaInfo)ret;
+#endif
+    return (CGImageAlphaInfo)0;
 }
 
 /**
  @Status Interoperable
  @Notes
 */
 size_t CGBitmapContextGetBitsPerComponent(CGContextRef context) {
+#if 0
     CGImageRef imageRef = context->Backing()->DestImage();
```

Comment:
nit: I don't think we'll miss anything but since these are technically being converted to stubs should we be changing the @status on these to stubs for proper tracking?